google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/BigtableChannelPrimer.java

```diff
@@ -41,27 +50,53 @@
 class BigtableChannelPrimer implements ChannelPrimer {
   private static Logger LOG = Logger.getLogger(BigtableChannelPrimer.class.toString());
 
-  private final EnhancedBigtableStubSettings settingsTemplate;
+  private final Metadata.Key<String> requestParams =
+      Metadata.Key.of("x-goog-request-params", Metadata.ASCII_STRING_MARSHALLER);
+  private final PingAndWarmRequest request;
+  private final Metadata metadata = new Metadata();
```

Comment:
This is fairly broken in a couple of ways:
1. it doesnt get populated...it gets shadowed in the constructor by a local variable
2. if it wasnt shadowed, then it would break because metadata instances cant be reused across multipple calls